app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageViewModel.kt

```diff
@@ -91,7 +91,7 @@ class ZimManageViewModel @Inject constructor(
   private val dataSource: DataSource,
   private val connectivityManager: ConnectivityManager,
   private val sharedPreferenceUtil: SharedPreferenceUtil
-) : ViewModel() {
+) : AndroidViewModel(context) {
```

Comment:
Also, it isn't recommended to use this `AndroidViewModel` as it is quite tightly coupled to android arch. reducing flexibility with our code that's why don't have it anywhere in the entire codebase of ours, so should be removed.

Reply:
@s-ayush2903 I was just playing around with that I thought I removed that. On it :)